executor/adapter.go

```diff
@@ -715,7 +715,7 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool) {
 			User:     userString,
 			DB:       sessVars.CurrentDB,
 			TableIDs: tableIDs,
-			IndexIDs: indexIDs,
```

Comment:
Do we need to update the name of `IndexIDs`?

Reply:
Thanks,  Done.